src/screens/OtherProfile/index.js

```diff
@@ -12,15 +13,14 @@ import {
   Text,
   TouchableNativeFeedback,
   TouchableOpacity,
-  View,
-  Pressable
+  View
 } from 'react-native';
 /* eslint-disable no-underscore-dangle */
 import {useNavigation} from '@react-navigation/core';
 import {useRoute} from '@react-navigation/native';
-import PropTypes from 'prop-types';
-import netInfo from '@react-native-community/netinfo';
+
 import ArrowUpWhiteIcon from '../../assets/icons/images/arrow-up-white.svg';
+import BioAndChat from './elements/BioAndChat';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `BioAndChat` component is added but `PropTypes` and `netInfo` are removed. Ensure that `PropTypes` is not required elsewhere in the code. If it is, you need to add it back. Also, if `netInfo` is not being used, it's a good practice to remove unused imports.
```diff
- import PropTypes from 'prop-types';
- import netInfo from '@react-native-community/netinfo';
+ import BioAndChat from './elements/BioAndChat';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->